.env.sample

```diff
@@ -242,3 +242,5 @@ AP_USERS_ORGANISATION_ID = ""
 CASSANDRA_HOST = ""
 CASSANDRA_PORT = ""
 CASSANDRA_DB = ""
+
+UPLOAD_PATH = ""
```

Comment:
@rakeshSgr - No need in env, put it in a file as a constant file

Reply:
resolved